src/commands/trace/trace.ts

```diff
@@ -152,7 +152,7 @@ export class TraceCommand extends Command {
   private getBaseIntakeUrl() {
     const site = process.env.DATADOG_SITE || process.env.DD_SITE || 'datadoghq.com'
 
-    return `https://${site}`
+    return `https://api.${site}`
```

Comment:
Should this get some kind of test?

Reply:
As a general rule I see that we do not really create unit tests for testing URLs themselves. Usually the developer runs manually some end to end tests